src/Animal.java

```diff
@@ -13,10 +13,10 @@ public Animal(int maxRunDistance, int maxSwimDistance) {
     }
```

Comment:
There is no need to have `int maxRunDistance; int maxSwimDistance;` variables in this class. All the checks should be in the respective subclasses